github-actions/start-release/templates/main_pr_body.md

```diff
@@ -1,4 +1,9 @@
 ### Notes
 - Merging next to main for the latest release
 - Please review the app version, generated release notes  
-and readme, and make any necessary updates
\ No newline at end of file
+and readme, and make any necessary updates
```

Comment:
unrelated to your change, but could you actually move this into the same line as above? 

I realized it doesn't need to be on a separate line?

Reply:
updated as suggested.